packages/web-react/src/components/Tooltip/demo/TooltipPlacements.tsx

```diff
@@ -141,13 +142,17 @@ const TooltipPlacements = () => {
           />
         </GridItem>
         <GridItem columnStart={2} rowStart={2}>
-          <TooltipWrapper UNSAFE_style={{ margin: '5rem auto' }}>
-            <DocsBox UNSAFE_className="px-900 py-900">
-              Click
-              <br /> the dots!
-            </DocsBox>
-            <Tooltip placement={placement as PlacementDictionaryType}>{placement}</Tooltip>
-          </TooltipWrapper>
+          <div className="mx-auto space-1200">
+            <Tooltip isOpen id="tooltip-placement" placement={placement as Placement} onToggle={() => {}}>
+              <TooltipTrigger elementType="div" UNSAFE_style={{ margin: '5rem auto' }}>
```

Comment:
can you use styleprops? Oh, we probably don't have one for rem. 🤔

Reply:
we don't

Reply:
It will be definitely handy to have the style props for margins and padding, or not?

Reply:
We have styleprops for margin, but only for space token values and auto. 
So we can do `marginX="auto"` but we would still need to do the 5rem by unsafestyle